components/Tournaments/MainExpandableRow/MainExpandableRow.tsx

```diff
@@ -90,7 +91,7 @@ export default function MainExpandableRow({ tournament }: { tournament: {} }) {
                 )}
               </div>
             </span>
-            <span>Missing Submitter</span>
+            <span>{tournament.submittedByUser?.player.username}</span>
```

Comment:
```suggestion
            <span>{tournament.submittedByUser?.player.username ?? 'Missing Submitter'}</span>
```
This still has a chance to be null